src/firstvoices/backend/models/category.py

```diff
@@ -2,15 +2,14 @@
 from django.db import models
 from django.utils.translation import gettext as _
 
-# FirstVoices
-from .sites import BaseSiteContentModel
+from firstvoices.backend.models.sites import BaseSiteContentModel
 
 
 class Category(BaseSiteContentModel):
     """Model for Categories."""
 
     # Fields
-    title = models.CharField(max_length=200)
+    title = models.CharField(max_length=75)
```

Comment:
Would it be useful to define this as a constant as well? (used here and in parts of speech)